cardano-node/src/Cardano/Node/Tracing/Era/Shelley.hs

```diff
@@ -161,10 +165,6 @@ instance LogFormatting (Conway.ConwayDelegPredFailure era) where
       , "amount" .= String (textShow credential)
       , "error" .= String "DRep already registered for the stake key"
       ]
-    Conway.WrongCertificateTypeDELEG ->
-      [ "kind" .= String "WrongCertificateTypeDELEG"
-      , "error" .= String "Wrong certificate type"
-      ]
 
```

Comment:
Has the underlying datatype removed this constructor?  And does InvalidPrevGovActionIdsInProposals replace it?